app/controllers/idv/socure/document_capture_controller.rb

```diff
@@ -81,6 +98,32 @@ def self.step_info
           end,
         )
       end
+
+      private
+
+      def handle_stored_result
+        if stored_result&.success? && selfie_requirement_met?
+          save_proofing_components(current_user)
+          extract_pii_from_doc(current_user, store_in_session: true)
+          flash[:success] = t('doc_auth.headings.capture_complete')
+          successful_response
+        else
```

Comment:
the user fails doc auth if the job to fetch and process the result is still in the queue?

Reply:
As discussed, we will add polling in a separate PR.